cst438_1/src/main/java/cst438_1/Cont.java

```diff
@@ -0,0 +1,42 @@
+package cst438_1;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+
+@Controller
+public class Cont {
+	@Autowired
+	MovieRepo movieRepo;
+	
+	@GetMapping("/movies/new")
+	public String createPerson( Model model) {
+		Movie movie = new Movie();
+		model.addAttribute("movie", movie);
+		return "movie_form";
+	}
+	
+	@PostMapping("/movies")
+	public String processMovieForm(@Valid Movie rating,
```

Comment:
you aren't adding the date-time before saving... in reference to your date time on your movie constructor is missing too.